common/src/main/java/org/apache/cassandra/sidecar/common/SidecarProps.java

```diff
@@ -0,0 +1,10 @@
+package org.apache.cassandra.sidecar.common;
+
+/**
+ * Properties maintained in common for all sub projects.
+ */
+public class SidecarProps
```

Comment:
do we need to introduce this class? Can the property be moved somewhere else?

Reply:
Moved that property to FileStreamer